docs/src/modules/components/AppSearch.js

```diff
@@ -1,18 +1,20 @@
 import * as React from 'react';
-import clsx from 'clsx';
 import useLazyCSS from 'docs/src/modules/utils/useLazyCSS';
 import useMediaQuery from '@material-ui/core/useMediaQuery';
-import { makeStyles } from '@material-ui/styles';
-import { alpha, useTheme } from '@material-ui/core/styles';
+import { alpha, styled, useTheme } from '@material-ui/core/styles';
 import Input from '@material-ui/core/Input';
 import SearchIcon from '@material-ui/icons/Search';
 import { handleEvent } from 'docs/src/modules/components/MarkdownLinks';
 import docsearch from 'docsearch.js';
 import { LANGUAGES_SSR } from 'docs/src/modules/constants';
 import { useUserLanguage, useTranslate } from 'docs/src/modules/utils/i18n';
 
-const useStyles = makeStyles(
-  (theme) => ({
+const StyledInput = styled(Input)({
+  color: 'inherit',
+});
+
+const RootDiv = styled('div')(({ theme }) => {
+  return {
     '@global': {
```

Comment:
Oops `@global` is JSS's directive. You need to convert these styles to `<GlobalComponent styles={} />`

Reply:
See for example - https://github.com/mui-org/material-ui/pull/27159/files#diff-30a7990d897b42037655d8e4826983520fef9dcebffc272c8af3437fcfe2655aR29

Reply:
Gotcha, thanks.

Reply:
fixed in [`914931a` (#27150)](https://github.com/mui-org/material-ui/pull/27150/commits/914931a917e4f133dfa776587cc888050f1c2aed)